src/main/java/seedu/address/logic/commands/CategorizeTagCommand.java

```diff
@@ -3,15 +3,17 @@
 import static java.util.Objects.requireNonNull;
 import static seedu.address.logic.parser.CliSyntax.PREFIX_TAG;
 
+import java.util.List;
+
 import seedu.address.logic.commands.exceptions.CommandException;
 import seedu.address.model.Model;
 import seedu.address.model.tag.Tag;
 import seedu.address.model.tag.TagCategory;
 
 
 /**
- * Implements command to categorize all occurrences of a tag.
- * Format: cattag t/TAG CATEGORY
+ * Implements command to categorize all occurrences of specified tags.
+ * Format: cattag t/TAG1 t/TAG2 ... t/TAGn CATEGORY
```

Comment:
Should we use this format for multiple tags or use t/TAG [t/MORE_TAGS] since the tags after TAG1 are optional? (t/TAG1 [t/TAG2...t/TAGn] maybe?